### ISSUE TITLE
Topical Area for Final Project

### ISSUE THREAD

**Issue by AmyTrumpower** [DS4PS/cpp-524-sum-2021#5]
Good afternoon Dr. Selby,

Would it be acceptable to use a program that revolves around foster/shelter dogs and adoptability? So far, my classes have all focused on social-work type programs that are concerned with people. 

Edit: Still thinking about it, maybe an environmental program or other animal-based program. Bottom line, does the program have to be a social focused program?

Thank you for your time,
Amy

**Comment by lecy** [comment]
Yes, you are encouraged to identify programs or policies within your area of interest. Animal welfare, environmental action, or the impact of arts on communities are all great topics. As long as you can characterize the program as some sort of 'intervention' that is expected to have impact. 

The one piece of advice is to select a narrow intervention. Try to avoid an organization that provides a large catalog of services such as nutritional support, financial literacy, life-coaching, job training, etc. Especially if each client receives a tailored list of services.

Also don't try to evaluate a collective impact endeavor like the Paris Climate Accord. You would need a PhD in climate science to create a theory of change regarding interventions that are supposed to impact a complex system like climate since there are so many feedback mechanisms that play out over the time-frame needed to observe the impact of the Accord. 

If you are keen on working with a large organization with comprehensive services (definitely use the assignment to learn about organizations that interest you) I would recommend picking one program from their catalog and focus on building an evaluation for that particular intervention. 

No matter how simple the program looks at the beginning you will see that things start to get complicated once you explore all of sampling and attrition challenges, the competing hypotheses, and potential threats to validity. 

So the topic is much less important than the scope of the program you select. 

@Dselby86

**Comment by Dselby86** [comment]
To add to what Dr. Lecy said.  The most important part of this project is to select a very clear and definable intervention with a limited number of possible treatments and clear outcomes.  However, you can be creative with your intervention. For instance, if you were interested in examining the effect of a program for adopting/fostering dogs you could propose an intervention where the shelter creates social media accounts/posts for each animal.  

But in thinkings about this project, start with what outcome an organization is aiming for, and then work backward to figure out a potential intervention that will achieve that outcome.  Once you have that solid foundation, the project will get surprisingly complex as you have to think about all the challenges that come from trying to measure that effectiveness of an intervention.

**Comment by AmyTrumpower** [comment]
Thank you both, I appreciate the feedback and the advice. This gives me solid foundation to start with. And I hear you about narrowing the focus.

Take care.
Amy

**Comment by mtwelker** [comment]
I appreciate that advice, Dr. Selby and Dr. Lecy.  Following up on that, can we select a program that is already operational?  For example, I wanted to look at cash payments to help homeless teens stay in school and graduate from high school.  Then I learned that a local agency for homeless teens is already doing this.  Can I still use this intervention for my project or do I need to come up with something new?  Would it be better to come up with a variation on what is already being done?  For example, to find the effect of doubling current payment amounts or implementing the program in other locations?  Thank you.

**Comment by lecy** [comment]
@mtwelker yes, existing programs are preferable because you can find some background information on them. You can pick a generic version (cash transfers to teens) or focus on a program currently run by a specific organization. The assignment is asking whether your evaluation design is sufficient for a program of that type, even if you are using a concrete example. You are welcome to introduce a twist like increasing payments if you like.

**Comment by mtwelker** [comment]
Thank you!
